tests/h/views/activity_test.py

```diff
@@ -917,14 +1096,48 @@ def group(factories):
     # Create some other groups as well, just to make sure it gets the right
     # one from the db.
     factories.Group()
-    factories.Group()
+    factories.OpenGroup()
 
     group = factories.Group()
 
     group.members.extend([factories.User(), factories.User()])
     return group
 
 
+@pytest.fixture
+def no_creator_group(factories):
+    group = factories.Group()
+    group.creator = None
+    group.members.extend([factories.User(), factories.User()])
+    return group
+
+
+@pytest.fixture
+def open_group(factories):
+    # Create some other groups as well, just to make sure it gets the right
+    # one from the db.
+    factories.Group()
```

Comment:
I'm still not clear on the purpose of creating these additional groups. Can you clarify?